lib/pf/services/manager/radiusd_child.pm

```diff
@@ -246,11 +283,49 @@ $options
 }
 EOT
     }
+
+    # Eduroam configuration
+    if ( @{pf::authentication::getAuthenticationSourcesByType('Eduroam')} ) {
+        my @eduroam_authentication_source = @{pf::authentication::getAuthenticationSourcesByType('Eduroam')};
+        my $tlrs1_server_address = $eduroam_authentication_source[0]{'tlrs1_server_address'};   # using array index 0 since there can only be one 'eduroam' authentication source ('unique' attribute)
+        my $tlrs2_server_address = $eduroam_authentication_source[0]{'tlrs2_server_address'};   # using array index 0 since there can only be one 'eduroam' authentication source ('unique' attribute)
+        my $tlrs_radius_secret = $eduroam_authentication_source[0]{'tlrs_radius_secret'};   # using array index 0 since there can only be one 'eduroam' authentication source ('unique' attribute)
+
+        $tags{'eduroam'} = <<"EOT";
```

Comment:
"proxy" in packetfence-cluster are the PacketFence servers themselves... i think that proxy.conf should refer to "outside" proxy servers... some that are not "PacketFence managed"
But I must say that I agree with you on the point that eduroam configuration should be all together in the same file (that would also include the clustering configuration)...
Let me rework that and I'll let you know